internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -26,8 +27,8 @@ import (
 	"io/fs"
 	"path/filepath"
 	"strings"
-	"testing/fstest"
 
+	"github.com/nlepage/go-tarfs"
```

Comment:
Hmm, we're probably way too early in the project lifecycle to care about this but I'll mention it anyways: but do we need to introduce a new direct dependency here?

Reply:
Fair question. I _did_ do a little bit of due diligence. It's actively maintained and has lots of tests. It also looks complex enough that if we tried to re-implement it, we'd probably have some maintenance overhead to contend with.

The one bug that was filed was fixed by the maintainer in 4 days.

TL;DR: I personally feel alright about this one.

Reply:
Thanks - that works for me. I don't think this is blocking anyways right now.